### PR TITLE
Add business income exclusion in Senate TCJA

### PR DESCRIPTION
**Pull request by codykallen**
This PR models two parts of the rules related to the exclusion of 17.4 percent of qualified business income, from the Senate's [Tax Cuts and Jobs Act](https://www.finance.senate.gov/imo/media/doc/11.9.17%20Chairman's%20Mark.pdf), pages 17-18. The proposal would allow 17.4% of qualified business income to be excluded from taxable income. The exclusion would phase out for personal services business income for taxable income over $75000 ($150k for married filing jointly). Since we cannot distinguish personal services business income from other business income, I have not modeled the phase-out. I also discuss this aspect in #1650. 

The exclusion rate for qualified business income is modeled by `_PT_exclusion_rt`. 
I also model an unusual rule from the bill:

> In the case of a taxpayer who has qualified business income from a partnership or S corporation, the amount of the deduction is limited to 50 percent of the W-2 wages of the taxpayer. 

This is modeled by `_PT_exclusion_limit`. 

@MattHJensen @martinholmer @jdebacker @feenberg @andersonfrailey @hdoupe @Amy-Xu @GoFroggyRun

### CODE REVIEW

File: taxcalc/current_law_policy.json [hunk 1]
```diff
@@ -3093,6 +3093,50 @@
...
+        "boolean_value": false,
+        "integer_value": false,
+        "value": [0.0],
+        "range": {"min": 0.0, "max": 1.0},
+        "out_of_range_minmsg": "",
+        "out_of_range_maxmsg": "",
+        "out_of_range_action": "stop"
+    },
+
+    "_PT_exclusion_limit": {
```
Comment (MattHJensen) [inline taxcalc/current_law_policy.json]: Could this be _PT_exclusion_wage_limit ?

**Comment by MattHJensen** [comment]
@codykallen, should this still have the WIP label (i.e., should it wait for review? )

**Comment by MattHJensen** [comment]
This looks good to me other than the one potential parameter naming issue.

File: taxcalc/current_law_policy.json (hunk 1 above)
Comment (codykallen) [reply taxcalc/current_law_policy.json]: That would be fine.

File: taxcalc/current_law_policy.json [hunk 2]
```diff
@@ -3093,6 +3093,50 @@
...
+        "row_var": "FLPDYR",
+        "row_label": ["2013"],
+        "start_year": 2013,
+        "cpi_inflated": false,
+        "col_var": "",
+        "col_label": "",
+        "boolean_value": false,
+        "integer_value": false,
+        "value": [9e99],
+        "range": {"min": 0.0, "max": 9e99},
```
Comment (andersonfrailey) [inline taxcalc/current_law_policy.json]: I'm a little confused about this parameter. The description says it's a fraction so shouldn't the value range be 0.0-1.0?

Comment (MattHJensen) [reply taxcalc/current_law_policy.json]: I think it's ok for this to be greater than 1. For example, you could limit PT income to 200% of wages.

Comment (codykallen) [reply taxcalc/current_law_policy.json]: When applied in the TCJA, it should be applied as a fraction. However, in the default state, it shouldn't apply at all. Thus the default value of 9e99, which makes the restriction inoperative unless deliberately changed.

Comment (andersonfrailey) [reply taxcalc/current_law_policy.json]: Ah gotcha. Thanks for the clarification.

**Comment by MattHJensen** [comment]
Looks good. Thanks for the contribution @codykallen and review @andersonfrailey.

**Pull request merged**